src/config/locales/en-US.json

```diff
@@ -131,7 +131,8 @@
     "open_orders":"Open Orders",
     "conversions_requested":"Conversions Requested",
     "expires":"expires",
-    "pending_requests":"Pending Requests"
+    "pending_requests":"Pending Requests",
+    "boost_account": "Boost"
```

Comment:
UI does not seem just here.... perhaps make `Boost` all caps like `GET POINTS` it can even be `GET BOOST`, asking feruz in group as well.... verified by feruz
![Screenshot 2022-08-15 at 12 36 14 PM](https://user-images.githubusercontent.com/6298342/184594849-3a89284b-a352-4f75-9fa8-20dd2966c644.png)